examples/react-client/fishjam-chat/src/components/DevicePicker.tsx

```diff
@@ -75,19 +75,19 @@ const DeviceSelect: FC<DeviceSelectProps> = ({ device }) => {
       <div className="flex justify-between">
         <Button
           onClick={async () => {
-            await device.toggle();
+            await device.toggleDevice();
           }}
           title="Stops and starts the physical device"
         >
-          Toggle Device
+          Toggle Device (start/stop)
         </Button>
         <Button
           onClick={async () => {
-            await device.toggle("soft");
+            await device.toggleMute();
           }}
           title="Disables or enables the device. Starts the device if it is stopped"
         >
-          Toggle Device (soft)
+          Toggle Device (mute/unmute)
```

Comment:
**question:** what do you think about _Toggle device / Toggle mute_?

Reply:
Toggle mute to me sounds very weird and I would assume is incomprehensible to anyone who hasn't had experience with our code

Reply:
fair